src/platform/PICO/system.c

```diff
@@ -114,8 +115,11 @@ void systemInit(void)
 
 void systemResetToBootloader(bootloaderRequestType_e requestType)
 {
-    UNUSED(requestType);
-    //TODO: implement
+    if (requestType == BOOTLOADER_REQUEST_FLASH) {
```

Comment:
(`switch (requestType) {'  ? )

Reply:
I wasn't going to implement the other items... as they default to a reboot... but I'll add the switch as I decided to put them in the if anyway